### DISCUSSION TITLE
Backport text command - Set labels to PRs

### DISCUSSION THREAD

**Discussion by EduPonz** [Mergifyio/mergify#5105] [Q&A]
Hi everyone,

We have this workflow were we develop against `main` and backport things to older supported versions using your this amazing tool. Our CI infrastructure is prepared to do different actions depending on the labels set on a PR when it's opened (in some cases we can avoid running tests, etc). When backporting PR which have these kinds of labels using the `backport` text command on a comment, we would like the ability to somehow keep the labels in the backport PRs openned by Mergify or even specify which labels should be added to the new PRs. I've seen that this is already possible with the Github action, so maybe adding this feature to the text command is not that difficult.

**Comment by jd** [comment]
Hi @EduPonz !

1. You could set labels to always add by using the `default` section of the configuration, but I doubt this would be useful.

2. You could leverage a GHA to copy the label from the original PR. We have something similar for copying labels in the merge queue: https://github.com/Mergifyio/gha-mergify-merge-queue-labels-copier
